lib/rubocop/cop/rspec/subject_stub.rb

```diff
@@ -73,72 +77,38 @@ class SubjectStub < Cop
             } ...)
         PATTERN
 
-        def on_block(node)
-          return unless example_group?(node)
+        def on_top_level_describe(node, _args)
+          @explicit_subjects = find_all_explicit_subjects(node.parent)
```

Comment:
why you would start from node.parent? You may end up getting a subject from another top-level describe

Reply:
I believe I will not. `node` here is `send` and parent is `block` in which this send is wrapped.

```ruby
s(:block,
  s(:send, nil, :describe,
    s(:const, nil, :Foo)),
  s(:args),
  s(:begin,
    s(:block,
      s(:send, nil, :subject,
        s(:sym, :foo)),
      s(:args),
      s(:send,
        s(:send, nil, :described_class), :new)),
...
```

Reply:
I see. I forgot top_level_describe looks for a send node. It's weird to me